functions/index.js

```diff
@@ -86,7 +86,9 @@ exports.sendSnackChat = functions.https.onCall((data, context) => {
   return authenticateUser(context.auth, () => {
     const tempFileName = '/tmp/snackchat.jpg';
     const fileName = `snackchat/${crypto.randomBytes(20).toString('hex')}.jpg`;
-    const bucket = admin.storage().bucket('gs://snackchat');
+    const snackchatUrl =
```

Comment:
Can we change the name of this to `snackchat.storageUrl`? I think the `slack` bit is confusing but maybe it was a typo?